docs/core/landmarks.md

```diff
@@ -10,3 +10,15 @@ for (bidirectional) A* and are done. It is important that although a weight
 **approximation** is used the landmarks algorithm is still not heuristic per
 default. You can make it faster and more heuristical behaving when you set
 the epsilon parameter lower than 1.
+
+For the world wide case we limit the routing in a certain way for landmarks.
+You can only route within the EU, Africa and Asia. The other algorithms are
```

Comment:
For routing requests using the landmark algorithm it is not possible to cross the borders between EU, Africa, and Asia?

Might be a bit more understandable? Or generally stating that you cannot route accross these borders using the landmark algorithm. The above sounds like routing in USA wouldn't be possible at all.